AUTHORS

```diff
@@ -9,6 +9,7 @@ Contributors
 
 Abhishek Patel
 Alan Crosswell
+Alejandro Mantecon Guillen
```

Comment:
Since this is only a doc change, feel free to remove my name from AUTHORS. I just added it because it was in the PR checklist.

Reply:
Documentation is as important as code!